scripts/release/index.ts

```diff
@@ -0,0 +1,77 @@
+import * as prettier from "prettier";
+import { readFile, writeFile } from "fs/promises";
+import minimist from "minimist";
+
+interface Args {
+  version: string | undefined;
+  emoji: string | undefined;
+}
+
+const args = minimist<Args>(process.argv.slice(2));
+
+main(args.version, args.emoji).catch((error) => {
```

Comment:
Wie roept `main` waarvandaan aan?

Reply:
Dat gaat met

```
yarn tsx scripts/release --version <version> --emoji <emoji>
```

maar ik heb in de root package.json een script gemaakt, en nu kan het zo:

```
yarn release --version <version> --emoji <emoji>
```

Reply:
Heb dit commando lokaal gebruikt:

`yarn release --version 1.2.3 --emoji <0001f9c3>`

En dan valt me op dat de datum YYYY-MM-dd ipv dd-MM-YYYY is. Vinden we dat erg?

Reply:
Nee, dat vinden we helemaal niet erg. We zijn een keertje overgestapt naar de Nederlandse notatie, terwijl keepachangelog.org (de standaard die wij gebruiken) de ISO date voorschrijft.

Vergeten mee te geven. Goed gezien weer

Reply:
In DSO Slack met Eric overeengekomen dat dit de way is